src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2126,6 +2145,23 @@ export class ResultsViewPageStore {
         }
     });
 
+    @computed get geneticEntities():GeneticEntity[]|undefined {
+        if (this.genes.isComplete && this.genesets.isComplete) {
+            let res: GeneticEntity[] = [];
+            for (const gene of this.genes.result) {
+                res.push({geneticEntityName: gene.hugoGeneSymbol, geneticEntityType: "gene", 
```

Comment:
always enums are better than strings, enums keep things clear, organized, and protect against typos. so lets do `enum GeneticEntityType { GENE, GENESET}` and use that for GeneticEntity.geneticEntityType